src/uwtools/tests/test_schemas.py

```diff
@@ -456,10 +455,116 @@ def test_schema_rocoto_workflow_cycledef():
 # sfc-climo-gen
 
 
+@fixture
+def sfc_climo_gen_fcstprop():
```

Comment:
I'm not sure about the `fcstproop` part of this name: I'm guessing it came from the `fv3_fcstprop` fixture, but I'm trying to recall why even that was a good name: Why not just `fv3_prop`, for example? I wonder if that name evolved from when the `fv3` driver was just called `forecast`... Anyway, unless it makes some sense to you, I don't see why we'd apply the term `fcst` to `sfc_climo_gen`, which has nothing to do with forecasting.

Similarly for `chgres_cube_fcstprop()`.

Reply:
You know, thats a good point - I should be making a habit of questioning existing objects. I'll ponder if there's still a better name

Reply:
I went and changed them all to `prop` , no need to make it any more complicated than necessary